screens/CreateMerchantScreen.tsx

```diff
@@ -115,18 +116,24 @@ export default function CreateMerchant({ navigation }: RootStackScreenProps<'Cre
         }
     }
 
+
     function RequiredField({ input }: { input: string }) {
         return (
             (!check || input) ? (
                 <></>
             ) : (
-                <Text style={styles.alert}>this field is required</Text>
+                <Text style={styles.alert}>This field is required</Text>
             ))
     }
 
+    function updatedText() {
+        setDisabledButton(false);
+        setValidMerchant(true);
```

Comment:
What you using this "validMerchant" state for and why do you set it to true very text change?